config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -386,6 +386,8 @@ periodics:
       - --test-cmd-args=--report-dir=/workspace/_artifacts
       - --test-cmd-args=--testconfig=testing/density/config.yaml
       - --test-cmd-args=--testconfig=testing/load/config.yaml
+      # TODO(https://github.com/kubernetes/kubernetes/issues/74213): Set everywhere if it works
+      - --test-cmd-args=--enable-prometheus-server
```

Comment:
Discussed offline - the current setup will not work with kubemark.

Reply:
Done.